docs/01-overview/02-main-areas/service-management/smgt-12-hb-bind-addons.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Bind addons
```

Comment:
same here, is this instructional or explanatory?

Reply:
Changed to `Addons for bindable plans`